readme.md

```diff
@@ -82,7 +100,15 @@ var Panda = function(name, age) {
 
 Your Answer:
 ```text
-
+eat_bamboo: function() {
+  var self = this;
+  var data = {
+    num_bamboo_eaten: self.num_bamboo_eaten + 1,
+  };
+  self.panda.update(data).then(function() {
+    self.render();
+  });
+},
```

Comment:
:+1:

We just asked you to increase `num_bamboo_eaten`.  No need for the mongoose model update.